docs/openapi.json

```diff
@@ -1468,6 +1542,50 @@
                     }
                 }
             },
+            "models.AdvisoryMetadata": {
```

Comment:
There is some mistake this object should not be used in API it's model for object-relation mapping.